Components/buttons/Button.html

```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Buttons</title>
+    <link rel="stylesheet" href="Button.css">
+    <link rel="stylesheet" href="/style.css">
+    <link rel="stylesheet" href="">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+
+<body>
+    <nav class="nav-body">
+        <ul class="list-col-left">
+            <li><a class="atribute" href="/">Home</a></li>
+            <li><a class="atribute" href="./Components/Avatar/Avatar.html">Documentation</a></li>
+            <ul class="list-col-right">
+                <li><a class="atribute" href="#"><i class="fa fa-github"></i></a></li>
+                <li><a class="atribute" href="#"><i class="fa fa-linkedin"></i></a></li>
+                <li><a class="atribute" href="#"><i class="fa fa-twitter"></i></a></li>
+            </ul>
+        </ul>
+    </nav>
+    <div class="container">
+        <div class="sideBar">
+            <h1 class="heading">components</h1>
+            <ul class="CompoItems">
+                <li class="componentlist"><a href="/Components/Avatar/Avatar.html">Avatar</a></li>
+                <li class="componentlist"><a href="/Components/Badge/Badge.html">Badge</a></li>
+                <li class="componentlist"><a href="/Components/Alert/Alert.html">Alert</a></li>
+                <li class="componentlist"><a href="./Button.html">Button</a></li>
+                <li class="componentlist"><a href="">Card</a></li>
+                <li class="componentlist"><a href="">Image</a></li>
+                <li class="componentlist"><a href="">Input</a></li>
+            </ul>
+        </div>
+        <div class="discriptionBar">
+            <h1 class="margin2rem">Button</h1>
```

Comment:
There should be only one `h1` per page

Reply:
thank you ..i didn't notice such thing but i'll resolve it